pom.xml

```diff
@@ -43,6 +43,7 @@
 
     <properties>
         <gridsuite-dependencies.version>25</gridsuite-dependencies.version>
+        <jaxb-api.version>2.4.0-b180830.0359</jaxb-api.version>
```

Comment:
This should be moved to the gridsuite dependencies to be shared by all services

Reply:
In powsybl-ws-dependencies instead ?
As there are also swagger pbs in network-conversion-server and single-line-diagram-server, who don't depend on gridsuite-dependencies ...